tests/measure/pos/Bag.hs

```diff
@@ -0,0 +1,27 @@
+module Bag where
```

Comment:
All these "new files" are just renames of old ones.  I'm not sure they were deleted in this commit, which is why they appear as "new"...